README.md

```diff
@@ -37,6 +37,13 @@ git clone git@github.com:SparkPost/wordpress-sparkpost.git ~/src/wordpress/wp-co
 
 * Activate the plugin from admin panel
 
+## Running Tests
+* Follow the steps in [wp-cli](http://wp-cli.org/docs/plugin-unit-tests/)
```

Comment:
Do we need to generate the plugin test files? Shouldn't they already be there from the first setup?